sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -69,11 +71,19 @@ export interface CertificateProperties extends ParsedKeyVaultEntityIdentifier {
     readonly x509Thumbprint?: Uint8Array;
 }
 
+// Warning: (ae-forgotten-export) The symbol "CertificatesClientInterface" needs to be exported by the entry point index.d.ts
+// 
 // @public
-export class CertificatesClient {
+export class CertificatesClient implements CertificatesClientInterface {
     constructor(url: string, credential: TokenCredential, pipelineOrOptions?: ServiceClientOptions | NewPipelineOptions);
     // Warning: (ae-forgotten-export) The symbol "BackupCertificateResult" needs to be exported by the entry point index.d.ts
     backupCertificate(name: string, options?: RequestOptionsBase): Promise<BackupCertificateResult>;
+    beginCertificateCreate(name: string, certificatePolicy: CertificatePolicy, createCertificateOptions?: CreateCertificateOptions, pollerOptions?: {
+        manual?: boolean;
+        intervalInMs?: number;
```

Comment:
the design guidelines call this `pollInterval` [1] but it definitely needs the `InMs` suffix, and `pollIntervalInMs` seems so verbose...I'll ask @bterlson about it

[1] https://azure.github.io/azure-sdk/typescript_design.html#ts-lro-support-options

Reply:
intervalInMs was suggested by Brian 🌞